packages/web-components/src/message-bar/spec.md

```diff
@@ -0,0 +1,98 @@
+# MessageBar
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31284&commitId=6bda335ce6435df6f3ca7ba475e505ea828e6725&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps.normal.chromium.png|1| Changed|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>